app/controllers/workorders_controller.rb

```diff
@@ -1,4 +1,17 @@
 class WorkordersController < ApplicationController
+  def index
+    @workorder = Workorder.all
+  end
+
+  def update
+    @workorder = Workorder.find(params[:id])
+    if params[:status] != ''
```

Comment:
Now that I think about it, I believe we can use `.blank?` See if that works the same way.